app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -114,7 +114,8 @@ def self.params_for_create
   end
 
   def self.verify_credentials(args)
-    !!raw_connect(args.dig("endpoints", "default")&.slice("server", "port", "token")&.symbolize_keys)
+    kubevirt = raw_connect(args.dig("endpoints", "default")&.slice("server", "port", "token")&.symbolize_keys)
+    kubevirt&.valid? && kubevirt&.virt_supported?
```

Comment:
The verification logic was kind of split between the kubernetes and kubevirt repos, kubevirt was doing `#valid?` and kubernetes was doing `#virt_supported?` [[ref]](https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/457/files#diff-1fe0d067134e4f6092c7cf36d13501a6d283dcc890e7279c0de4059af6a49de5L817)

Seemed much simpler to just move all verify logic into this method